From: Dave Jones <davej@redhat.com>
Date: Mon, 27 Jan 2014 23:11:09 -0500
Subject: [PATCH v3 203/474] i40e: Add missing braces to
 i40e_dcb_need_reconfig()
Patch-mainline: v3.14-rc1
Git-commit: 3d9667a9e1e463c107cb47a810ef7e85d9a31e62
References: bsc#909484 FATE#317397

Indentation mismatch spotted with Coverity.
Introduced in 4e3b35b044ea ("i40e: add DCB and DCBNL support")

Signed-off-by: Dave Jones <davej@fedoraproject.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/intel/i40e/i40e_main.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -4435,9 +4435,10 @@ bool i40e_dcb_need_reconfig(struct i40e_
 	/* Check if APP Table has changed */
 	if (memcmp(&new_cfg->app,
 		   &old_cfg->app,
-		   sizeof(new_cfg->app)))
+		   sizeof(new_cfg->app))) {
 		need_reconfig = true;
 		dev_info(&pf->pdev->dev, "APP Table change detected.\n");
+	}
 
 	return need_reconfig;
 }
